PurchasesCoreSwift/Misc/SystemInfo.swift

```diff
@@ -51,27 +59,27 @@ import AppKit
         return receiptURLString.contains("sandboxReceipt")
     }
 
-    @objc public static var frameworkVersion: String { // TODO: automate the setting of this, if it hasn't been.
+    static var frameworkVersion: String { // TODO: automate the setting of this, if it hasn't been.
         return "3.13.0-SNAPSHOT"
```

Comment:
addressing the comment here: this has been automated, but I'm guessing we need to update the fastlane lane's references to this file

Reply:
#767